### PR TITLE
Feat(audio): Implement Recorder

### PR DESCRIPTION
**Pull request by ryandeyo1129**
- implement audio recordings

### CODE REVIEW

File: .eslintcache [hunk 1]
```diff
@@ -0,0 +1 @@
...o context time.\n    scheduleNote(currentNote, nextNoteTime);\n    nextNote();\n  }\n  timerID = window.setTimeout(scheduler, lookahead);\n}\n\n// UPDATE VALUES\nexport const resetCounters = () => {\n  nextNoteTime = 0.0;\n  startTime = audioCtx.currentTime + 0.200;\n};\n\nexport const stop = () => {\n  currentNote = 0;\n  window.clearTimeout(timerID);\n}\n\nexport const updatePadMap = (map: any) => padMap = map;\n\nexport const updateTempo = (bpm: number) => tempo = bpm;","/Users/f1v/Work/drumroll-please-frontend/src/recorder/Recorder.tsx",[],"/Users/f1v/Work/drumroll-please-frontend/src/components/CreateNewModule.tsx",["36"],{"ruleId":"37","replacedBy":"38"},{"ruleId":"39","replacedBy":"40"},{"ruleId":"41","severity":1,"message":"42","line":58,"column":43,"nodeType":"43","messageId":"44","endLine":58,"endColumn":49},{"ruleId":"41","severity":1,"message":"45","line":2,"column":10,"nodeType":"43","messageId":"44","endLine":2,"endColumn":20},"no-native-reassign",["46"],"no-negated-in-lhs",["47"],"@typescript-eslint/no-unused-vars","'padMap' is assigned a value but never used.","Identifier","unusedVar","'withRouter' is defined but never used.","no-global-assign","no-unsafe-negation"]
```
Comment (asooge) [inline .eslintcache]: I don't think we want to include this file.

Comment (ryandeyo1129) [reply .eslintcache]: I don't know why that keeps showing up, I thought I had removed it.

Comment (ryandeyo1129) [reply .eslintcache]: ok that should be gone this time.

File: src/App.tsx [hunk 2]
```diff
@@ -41,6 +43,7 @@ function App() {
               <Route path="/loops/:id" component={Sequencer} />
             </Switch>
           </Container>
+          <Recorder />
```
Comment (asooge) [inline src/App.tsx]: We probably want to move this elsewhere, perhaps Sequencer.tsx

**Comment by asooge** [comment]
LGTM. We probably want to relocate the `<Recorder />` component but the functionality is good.

**Comment by asooge** [comment]
LGTM

**Review by alex-f1v — APPROVED** [review]
LGTM

**Pull request merged**